spec/css/custom_properties/trailing_whitespace.hrx

```diff
@@ -7,10 +7,28 @@
   --before-closing-brace: value
 }
 
-<===> output.css
+<===> scss/output.css
 .trailing-whitespace {
   --space: value ;
   --tab: value	;
   --newline: value ;
   --before-closing-brace: value ;
 }
+
+<===>
+================================================================================
+<===> sass/input.sass
```

Comment:
If we're adding new tests, we should make them style-guide-conformant even if they're largely copies of old tests. Bonus points to make the old tests conformant as well.